upb/port_def.inc

```diff
@@ -181,3 +181,26 @@ int msvc_vsnprintf(char* s, size_t n, const char* format, va_list arg);
 #else
 #define UPB_NAN (0.0 / 0.0)
 #endif
+
+// User code should use macros instead of functions.
```

Comment:
what do you mean here? I would think user code should never use UPB macros.

Reply:
Bad copy/paste, removed.